app/assets/stylesheets/views/listings.scss

```diff
@@ -91,3 +91,9 @@
     display: none;
   }
 }
+
+.listings-modal {
+  .crayons-modal__box__body {
+    padding: 0;
```

Comment:
Question (non-blocking): Curious why we're changing the default styling of the modal for Listings.

Reply:
I'm not 100% but AFAIR it's because Listings modals have unique design (probably too unique) --> that grey background going from edge to edge would not be possible with original padding.

But I don't think that particular styling was introduce in current PR - I believe it's just moved from somewhere else.

Also, we could do some refactor at some point and add a prop to modal allowing no padding (i think it might be a common scenario)

```
<Modal noPadding={true}>
```

Reply:
@ludwiczakpawel is 100% right - the listings modal is another that doesn't quite conform to our standard crayons modal, so we have to remove the default padding. Adding the suggested `noPadding` prop seems like a good solution. I could raise a small issue for this 👍

Reply:
#12664